docs/ui/components/ContentSpotlight/index.tsx

```diff
@@ -32,93 +30,73 @@ export function ContentSpotlight({
   const isVideo = !!(url || file);
+  const maxWidthClass = maxWidth && `max-w-[${maxWidth}px]`;
 
   return (
     <figure
       className={mergeClasses(
-        'text-center py-2.5 my-5 rounded-lg',
+        'text-center py-2.5 my-5 rounded-lg cursor-pointer',
         containerClassName,
         !isVideo && 'bg-subtle'
       )}
       onClick={() => {
         if (typeof controls === 'undefined' && !forceShowControls) {
           setForceShowControls(true);
         }
-      }}
-      onMouseEnter={() => setHover(true)}
-      onMouseLeave={() => setHover(false)}
-      style={hover ? { cursor: 'pointer' } : undefined}>
+      }}>
       {src ? (
         <LightboxImage
           src={src}
           alt={alt}
-          style={style}
-          className="inline rounded-md transition-opacity duration-default ease-in-out hover:opacity-80"
+          className={mergeClasses(
+            'inline rounded-md transition-opacity duration-default ease-in-out hover:opacity-80',
+            maxWidthClass
+          )}
         />
       ) : isVideo ? (
         <VisibilitySensor partialVisibility>
           {({ isVisible }: { isVisible: boolean }) => (
-            <div className={getInitialMarginBottom(spaceAfter)}>
-              <div className="relative w-full h-[400px] bg-palette-black rounded-lg overflow-hidden">
-                <ReactPlayer
-                  url={isVisible ? url || `/static/videos/${file}` : undefined}
-                  className="react-player"
-                  width={PLAYER_WIDTH}
-                  height={PLAYER_HEIGHT}
-                  style={{
-                    outline: 'none',
-                    backgroundColor: '#000',
-                  }}
-                  muted
-                  playing={isVisible && !!file}
-                  controls={typeof controls === 'undefined' ? forceShowControls : controls}
-                  playsinline
-                  loop={loop}
-                />
-                <div
-                  className={`pointer-events-none absolute inset-0 transition-opacity duration-500 ${
-                    isVisible ? 'opacity-0' : 'opacity-70'
-                  } md:hidden`}
-                />
-              </div>
+            <div className="relative w-full h-[400px] bg-palette-black rounded-lg overflow-hidden">
+              <ReactPlayer
+                url={isVisible ? url || `/static/videos/${file}` : undefined}
+                className="react-player"
+                width={PLAYER_WIDTH}
+                height={PLAYER_HEIGHT}
+                muted
+                playing={isVisible && !!file}
+                controls={typeof controls === 'undefined' ? forceShowControls : controls}
+                playsinline
+                loop={loop}
+              />
+              <div
+                className={mergeClasses(
+                  'pointer-events-none absolute inset-0 transition-opacity duration-500 md:hidden',
```

Comment:
```suggestion
                  'pointer-events-none absolute inset-0 transition-opacity duration-500 max-md-gutters:hidden',
```

We should use the `max-*-gutters` scope variant for breakpoints adjustments, plain scopes are a side effect of Tailwind custom theme generation. The only valid use case for those are screen size classes `max-w-screen-md`.